### ISSUE TITLE
[FE] 10-10EZ - Facility Selection: Remove feature toggle [hca_use_facilities-API]

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#75196]
## Background
The static list used today is very outdated and causing issues for Veterans who are completing the 10-10EZ form.
We have connected to the Facilities API in lower level environments with success, and need to implement this functionality in Production.  This will allow a dynamic list of facilities to populate a dropdown menu, which will always be a current list and remove the manual maintenance from the 10-10 team.

Validations for EZ are handled through vets-json-schema. The validation checks the preferred facility against an enum (list) of facilities in the schema. Lighthouse supplies more facilities than we have in the enum, so any selected facility from lighthouse, that is not in the schema, will trigger a validation failure.

Once ticket #79441 is complete, we can do this update.

## Tasks

- [ ] Update vets-api to remove hca_use_facilities_API from the config/features.yml file
  - (instructions: https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Removingold/unusedfeaturetoggles)

## Acceptance Criteria
- Facility selection page on the 10-10EZ will use the current Facilities API to display the facilities within the state selected by the Veteran.

**Comment by hdjustice** [comment]
Validated with Josh Faulkner that there have not been any issues or failures due to the facilities API update completed August 26th.  We can move forward with removing the feature toggle

**Comment by hdjustice** [comment]
closing as a duplicate of #92316
